release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,17 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+
+[id="ocp-4-6-security"]
+=== Security
+
+[id="ocp-4-6-oauth-token-storage"]
+==== Secure OAuth token storage format
+
+OAuth access token and OAuth authorize token object names are now stored as non-sensitive object names.
+
+Previously, secret information was used as the OAuth access token and OAuth authorize token object names. When etcd is encrypted, only the value is encrypted, so this sensitive information was not encrypted.
+
```

Comment:
Note: for cluster installed prior to 4.6, ensure that old tokens (without a `sha256~` prefix) either expire eventually or are manually deleted in order to switch to a secure storage format.

Reply:
Added a note on this, thanks!